DEVELOPMENT-GUIDE.md

```diff
@@ -0,0 +1,85 @@
+# ACCP Development Guidance
+Correctness and code-safety is paramount in the Amazon Corretto Crypto Provider.
+
+The purpose of this guide is not to provide all information needed for development within ACCP.
+Instead, it is intended to provide a quick introduction to the most important (and ACCP-specific) components to help developers find and learn what is most important.
+
+# Development Principles
+
+In decreasing order of importance:
```

Comment:
Delete. This is typical and expected.

Reply:
Unlike everything else (which I'll take), I really want to keep this. Principles are not always explicitly ordered and it is useful to call out when they are.